snippets/csharp/System.Collections.Concurrent/ConcurrentBag/concbag.cs

```diff
@@ -36,7 +36,7 @@ static void Main()
                 if (cb.TryTake(out item))
                 {
                     Console.WriteLine(item);
-                    itemsInBag++;
+                    Interlocked.Increment(ref itemsInBag);
```

Comment:
This needs the following using statement to compile correctly:

```
using System.Threading;
```

Once you add that, we'll :shipit:

Reply:
Thank you!  I updated.